src/screens/settings/components/other-settings.tsx

```diff
@@ -0,0 +1,100 @@
+import { NavigationContainerRef, useNavigation } from '@react-navigation/native';
+import { Linking, Pressable, View } from 'react-native';
+import { Iconify } from 'react-native-iconify';
+
+import { Typographies } from '~/shared/components/typographies';
+import { useStyles } from '~/shared/hooks/use-styles';
+import { useTranslate } from '~/shared/hooks/use-translate';
+import { RootStackParamList } from '~/shared/navigation';
+import { createStylesheet } from '~/shared/styles/create-stylesheet';
+
+interface ButtonProps {
+  title: string;
+  redirectTo: string;
+}
+
+export const OtherSettings = () => {
+  const styles = useStyles(getStyles);
+
+  const translate = useTranslate();
+
+  const buttons: ButtonProps[] = [
+    {
+      title: translate('settings.other.buttonTitles.bugReport'),
+      redirectTo: 'https://github.com/wanned/karmine-corp-app/issues/new',
+    },
+    {
+      title: translate('settings.other.buttonTitles.karmineCorpWebsite'),
+      redirectTo: 'https://www.karminecorp.fr/',
+    },
+    {
+      title: translate('settings.other.buttonTitles.karmineCorpTwitter'),
+      redirectTo: 'https://x.com/KarmineCorp',
+    },
+    {
+      title: translate('settings.other.buttonTitles.credits'),
+      // TODO: Add credits screen redirect
+      redirectTo: 'home',
+    },
+  ];
+
+  return (
+    <View style={styles.ButtonContainer}>
+      {buttons.map((button, index) => (
+        <Button key={index} title={button.title} redirectTo={button.redirectTo} />
+      ))}
+    </View>
+  );
+};
+
+const getStyles = createStylesheet((theme) => ({
+  ButtonContainer: {
+    marginTop: 12,
+    gap: 8,
+  },
+}));
+
+const Button = ({ title, redirectTo }: ButtonProps) => {
```

Comment:
Use an external component. Maybe add a `withArrow` prop on the secondary button. Then, we could pass `() => navigateOrOpenURL(redirectTo)` to the `onPress` prop